README.md

```diff
@@ -192,7 +192,11 @@ Install the Wazuh app
 
 ```
 cd /usr/share/kibana/
+<<<<<<< HEAD
```

Comment:
@frankeros This merge has introduced some unresolved merge conflicts here and on:

https://github.com/wazuh/wazuh-kibana-app/pull/3352/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R228

causing a broken README.md.

Reply:
#3365